<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugin::lunar::SettingsWidget</class>
 <widget class="QWidget" name="plugin::lunar::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Format</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="format_group">
     <property name="title">
      <string>Display Format</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="simple_rbtn">
        <property name="text">
         <string>Simple (正月十五)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="standard_rbtn">
        <property name="text">
         <string>Standard (丙午年 正月十五)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="full_rbtn">
        <property name="text">
         <string>Full (丙午年【马】正月十五)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
